src/Place/Models/PDB01MPlace/Contexts/PCT012MPlaceOne/ModelComponents/PMC01202PlaceInformation/PlaceInformation.tsx

```diff
@@ -51,21 +51,13 @@ export const PMC01202PlaceInformation = () => {
 					<Text style={styles.infoText}>{i18n.t('Opening Hours')}</Text>
 				</MaterialCommunityIcons>
 				{Array.isArray(place.openingHours) ? (
-					// TODO: https://github.com/Ayato-kosaka/spelieve/issues/377 PMC01202PlaceInformationのFlatList をmapに変更する
-					<FlatList
-						data={place.openingHours}
-						renderItem={(itemData) => {
-							const [day, time] = itemData.item;
-							return (
-								<View>
-									<Text style={styles.infoText}>
-										{day} {time}
-									</Text>
-								</View>
-							);
-						}}
-						numColumns={1}
-					/>
+					place.openingHours.map(([day, time]) => (
+						<View key={day}>
```

Comment:
あんまりロジック理解できていないけど、多分dayだと一意にならないよね？（day,time で一意かなと思った）
だから↓みたいにすると良いと思う
```suggestion
						<View key={`${day}${time}`}>
```